### ISSUE TITLE
Status bar on iOS bug

### ISSUE THREAD

**Issue by miquelferrerllompart** [ionic-team/ionic#13871]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[X] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[X] bug report
[ ] feature request

**Current behavior:**

On PWA over iOS, if you change the position of mobile (portrait -> landscape -> portrait) the content is overlapped for status bar.

**Expected behavior:**

The PWA on iOS always should respect status bar height and not overlap it.

**Steps to reproduce:**
1 - Open PWA on iOS, landscape mode
2 - When PWA is full dowloaded, change the position of iphone to landscape
3 - Change the position of iPhone to portrait
4 - You can see how th status bar safe zone is removed

**Related code:**
I think i have the all necessary meta-tags on my `src/index.html`
```html
<meta name="viewport" content="viewport-fit=cover, width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
<meta name="apple-mobile-web-app-status-bar-style" content="white">
```

**Other information:**

I also have the cordova-plugin-statusbar plugin.
This is my package.json
```json
{
  "name": "some-name",
  "version": "0.0.34",
  "author": "some-author",
  "homepage": "http://www.some-web.com/",
  "private": true,
  "scripts": {
    "clean": "ionic-app-scripts clean",
    "build": "ionic-app-scripts build",
    "ionic:build": "ionic-app-scripts build",
    "ionic:serve": "ionic-app-scripts serve",
    "watch": "ionic-app-scripts watch",
    "serve:before": "watch",
    "emulate:before": "build",
    "deploy:before": "build",
    "build:before": "build",
    "run:before": "build",
    "ionic:lint": "tslint --project tslint.json './src/**/*.ts' --exclude './src/shared/**'",
    "sass-lint": "sass-lint c app/config/.sass-lint.yml './src/**/*.scss' -v -q",
    "extract": "ngx-translate-extract --input ./src --output ./src/assets/i18n/template.pot --clean --sort --format pot && msgcat ./src/assets/i18n/template.pot ./src/assets/i18n/manual-strings.pot > ./src/assets/i18n/update.pot --use-first"
  },
  "config": {
    "ionic_source_map": "source-map",
    "ionic_webpack": "./src/config/webpack.config.js"
  },
  "dependencies": {
    "@angular/common": "5.2.0",
    "@angular/compiler": "5.2.0",
    "@angular/compiler-cli": "5.2.0",
    "@angular/core": "5.2.0",
    "@angular/forms": "5.2.0",
    "@angular/http": "5.2.0",
    "@angular/platform-browser": "5.2.0",
    "@angular/platform-browser-dynamic": "5.2.0",
    "@biesbjerg/ngx-translate-po-http-loader": "^2.0.1",
    "@ionic-native/app-version": "4.4.2",
    "@ionic-native/barcode-scanner": "4.4.2",
    "@ionic-native/call-number": "^4.5.2",
    "@ionic-native/camera": "4.4.2",
    "@ionic-native/clipboard": "4.4.2",
    "@ionic-native/core": "4.4.2",
    "@ionic-native/email-composer": "4.4.2",
    "@ionic-native/keyboard": "4.4.2",
    "@ionic-native/splash-screen": "4.4.2",
    "@ionic-native/status-bar": "^4.5.2",
    "@ionic/pro": "^1.0.14",
    "@ionic/storage": "2.1.3",
    "@ngx-translate/core": "8.0.0",
    "apollo-angular": "^1.0.0",
    "apollo-angular-link-http": "^1.0.1",
    "apollo-cache-inmemory": "^1.1.1",
    "apollo-client": "^2.0.3",
    "apollo-link": "^1.0.3",
    "apollo-link-error": "^1.0.1",
    "call-number": "^1.0.1",
    "chart.js": "^2.7.1",
    "cordova-android": "6.3.0",
    "cordova-clipboard": "^1.1.1",
    "cordova-ios": "4.5.2",
    "cordova-plugin-app-version": "^0.1.9",
    "cordova-plugin-camera": "^3.0.0",
    "cordova-plugin-compat": "^1.2.0",
    "cordova-plugin-device": "^1.1.7",
    "cordova-plugin-email": "^1.2.6",
    "cordova-plugin-ionic-webview": "^1.1.16",
    "cordova-plugin-splashscreen": "^4.1.0",
    "cordova-plugin-whitelist": "^1.3.3",
    "cordova-sqlite-storage": "^2.2.0",
    "graphql": "^0.11.7",
    "graphql-tag": "^2.5.0",
    "ionic-angular": "3.9.2",
    "ionic-plugin-keyboard": "^2.2.1",
    "ionicons": "3.0.0",
    "lodash": "^4.17.4",
    "mx.ferreyra.callnumber": "0.0.2",
    "ng2-charts": "^1.6.0",
    "ngx-pipes": "^2.0.5",
    "phonegap-plugin-barcodescanner": "^6.0.8",
    "rxjs": "^5.5.2",
    "sw-toolbox": "3.6.0",
    "zone.js": "0.8.18"
  },
  "devDependencies": {
    "@biesbjerg/ngx-translate-extract": "2.3.4",
    "@ionic/app-scripts": "^3.1.7",
    "@types/async": "2.0.31",
    "@types/graphql": "^0.11.6",
    "sass-lint": "^1.10.2",
    "tslint": "^5.8.0",
    "tslint-ionic-rules": "^0.0.11",
    "typescript": "2.4.2"
  },
  "cordovaPlugins": [
    "cordova-plugin-whitelist",
    "cordova-plugin-console",
    "cordova-plugin-statusbar",
    "cordova-plugin-device",
    "cordova-plugin-splashscreen",
    "ionic-plugin-keyboard",
    "ionic-plugin-app-version"
  ],
  "cordovaPlatforms": [
    "ios",
    {
      "platform": "ios",
      "version": "",
      "locator": "ios"
    }
  ],
  "description": "some description",
  "cordova": {
    "plugins": {
      "phonegap-plugin-barcodescanner": {
        "CAMERA_USAGE_DESCRIPTION": "BarcodeScanner",
        "NSCameraUsageDescription": "Cam for BarcodeScanner"
      },
      "cordova-plugin-camera": {
        "CAMERA_USAGE_DESCRIPTION": "BarcodeScanner",
        "NSCameraUsageDescription": "Cam for BarcodeScanner",
        "PHOTOLIBRARY_USAGE_DESCRIPTION": "BarcodeScanner"
      },
      "cordova-plugin-device": {},
      "cordova-plugin-email": {},
      "cordova-plugin-splashscreen": {},
      "cordova-plugin-statusbar": {},
      "cordova-plugin-whitelist": {},
      "ionic-plugin-keyboard": {},
      "cordova-sqlite-storage": {},
      "cordova-plugin-ionic-webview": {},
      "cordova-plugin-app-version": {},
      "cordova-clipboard": {},
      "mx.ferreyra.callnumber": {},
      "call-number": {}
    },
    "platforms": [
      "ios",
      "android"
    ]
  }
}

```
**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
cli packages: (/Users/miquelferrerllompart/.nvm/versions/node/v9.3.0/lib/node_modules)

    @ionic/cli-utils  : 1.19.1
    ionic (Ionic CLI) : 3.19.1

global packages:

    cordova (Cordova CLI) : 6.5.0

local packages:

    @ionic/app-scripts : 3.1.7
    Cordova Platforms  : android 6.4.0 ios 4.5.3
    Ionic Framework    : ionic-angular 3.9.2

System:

    Android SDK Tools : 26.1.1
    ios-deploy        : 1.9.2
    ios-sim           : 6.1.2
    Node              : v9.3.0
    npm               : 5.6.0
    OS                : macOS High Sierra
    Xcode             : Xcode 9.2 Build version 9C40b

Environment Variables:

    ANDROID_HOME : /Users/miquelferrerllompart/Library/Android/sdk

Misc:

    backend : pro
```

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us! Would you be able to provide a sample application via GitHub that demonstrates the issue you are having? Thanks for using Ionic!

**Comment by kensodemann** [comment]
Also please make sure you have followed all of the advice given here: https://blog.ionicframework.com/ios-11-checklist/

**Comment by miquelferrerllompart** [comment]
Hi,
Thanks for the quicky response!
I can make a litle repo to show the problem, but yestarday I investigated a litle more and I could see how another big sites have the same problem (Wego, Flipkart). Others, like twitter litle, has deactivated standalone mode, maybe cab be for that?
I think this is a iOS bug with PWA standalone mode.

**Comment by RHinderiks** [comment]
I have this same issue on a non pwa app. Hiding the statusbar to show full screen image. When rotated to landscape and then showing the statusbar the statusbar will overlap the header.

**Comment by malwatte** [comment]
We faced this kind of issue. We did not use statusbar and cordova-plugin-ionic-webview plugins. So the fix was to add statusbar (use git URL to install the plugin) and cordova-plugin-ionic-webview
ionic cordova plugin add https://github.com/apache/cordova-plugin-statusbar
ionic cordova plugin add cordova-plugin-ionic-webview
Then all worked fine.
Looks like UIWebview does not support safe regions well.
